ofrak_core/ofrak/model/data_model.py

```diff
@@ -43,26 +43,19 @@ def __repr__(self):
 @dataclass
 class DataPatchResult:
     range: Range
-    size_change: int
-    child_index: int
-    child_relative_position: DataRangePosition
 
 
 @dataclass
 class DataPatchesResult:
     data_id: bytes
     patches: List[DataPatchResult]
 
-    def get_size_change(self):
-        return sum(r.size_change for r in self.patches)
 
-
-@dataclass
+@dataclass(unsafe_hash=True)
```

Comment:
It makes the models hashable so they can be added to sets. This only is used in a scope where the models cannot change while the set is active, but since the models are an external-facing data structure, it might not be safe to include. I'll work it out of there.